README.md

```diff
@@ -83,11 +83,9 @@ Only one address can have this role.
 It can:
 
 - Revoke the pending timelock.
-- Revoke the pending guardian.
+- Revoke the pending guardian (in particular, it can revoke any submission of another guardian).
```

Comment:
```suggestion
- Revoke the pending guardian (which means it can revoke any attempt to change the guardian).
```
(optional) I think it's clearer, even though it is not using the word `submit`